analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -274,8 +274,16 @@ public void Execute_ClearsCapturesAfterBranching()
         }
 
         [TestMethod]
-        public void Execute_LocalScopeRegion_AssignDefaultBoolConstraint() =>
-            SETestContext.CreateVB(@"Dim B As Boolean : Tag(""B"", B)").Validator.ValidateTag("B", x => x.HasConstraint(BoolConstraint.False).Should().BeTrue());
+        public void Execute_LocalScopeRegion_Boolean_AssignDefaultBoolConstraint() =>
+            SETestContext.CreateVB(@"Dim Value As Boolean : Tag(""Value"", Value)").Validator.ValidateTag("Value", x => x.HasConstraint(BoolConstraint.False).Should().BeTrue());
```

Comment:
Add a test with an initializer to `True`

Reply:
That would not be default anymore. Assignment is a different story